website/docs/developer/development-guide/documentation/documentation-deployment.md

```diff
@@ -4,4 +4,73 @@ title: Deploying Documentation
 desc: Provides basic information on deploying ORCA documentation to nasa.github.io
 ---
 
-Information on deploying the documentation is TBD.
+The following sections provide information to ORCA users to create automated or
+manual releases on github. View the entire versioning and releases document 
+[here](https://github.com/nasa/cumulus-orca/blob/master/docs/release.md).
+
+## Automated Documentation Release on Github
+
+### Create a new ORCA release on github
+
+The release is automated in Bamboo, but the step must be manually started. If 
+you set the `RELEASE_FLAG` to `true` and the build steps passed, you will be 
+able to run the manual "Release" step in Bamboo.
+
+The CI release scripts will create a release based on the release version tag, 
+as well as uploading release artifacts to the Github release for the Terraform 
+modules provided by Cumulus. The Terraform release artifacts include:
+
+ * A multi-module Terraform .zip artifact containing filtered copies of the 
+   tf-modules, packages, and tasks directories for use as Terraform module sources.
+
+Just make sure to verify the appropriate .zip files are present on Github after 
+the release process is complete.
+
+### Merge the base branch into develop and master
```

Comment:
This section can probably be removed if we reference them to the release documentation. For the website display there is no merging that occurs.

Reply:
removed section